transport-native-epoll/src/main/c/io_netty_channel_epoll_Native.c

```diff
@@ -78,25 +78,25 @@ static int socketType;
 static const char* ip4prefix = "::ffff:";
 
 // util methods
-void throwRuntimeException(JNIEnv* env, char* message) {
+static void throwRuntimeException(JNIEnv* env, char* message) {
```

Comment:
These `throw*` methods seem to be a good candidate for inlining. Could you mark them inlined?